_topic_maps/_topic_map.yml

```diff
@@ -1349,18 +1349,10 @@ Topics:
       File: aws-load-balancer-operator-release-notes
     - Name: AWS Load Balancer Operator in OpenShift Container Platform
       File: understanding-aws-load-balancer-operator
+    - Name: Preparing an AWS STS cluster for the AWS Load Balancer Operator
+      File: installing-albo-sts-cluster
```

Comment:
Should the title here be "Preparing..."? In the preview build, the title of `installing-albo-sts-cluster` is "Installing the AWS Load Balancer Operator on a cluster that uses AWS STS". Seems mismatched.

Reply:
It definitely should be "preparing." The original author wrote this section who-knows-when, but it doesn't include any installing ALBO instructions. It just has instructions on how to prepare a cluster with STS so you can use ALBO.

I was trying to be minimal in my changes- I'm happy to change this to suit.

Reply:
This should be resolved as of the latest commit.